context/etcd-mgmt.go

```diff
@@ -0,0 +1,36 @@
+package context
```

Comment:
Why this needs to be in `context` package? You could have a separate package called etcd-mgmt and then have bootstrap.go.